packages/react/src/primitives/Loader/Loader.tsx

```diff
@@ -50,7 +50,7 @@ const LoaderPrimitive: Primitive<LoaderProps, 'svg'> = (
         x2="100%"
         y1="50%"
         y2="50%"
-        style={{ stroke: emptyColor }}
+        style={{ stroke: `${emptyColor}` }}
```

Comment:
```suggestion
       style={{ stroke: emptyColor }}
```

Reply:
We cannot do that because the native style prop doesn't accept `StyleTokens` type and we have to convert it into a string. Same for the others

Reply:
I see...Then, do you think it would be more explicit if we do a `emptyColor.toString()`